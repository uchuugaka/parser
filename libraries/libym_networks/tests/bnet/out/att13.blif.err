"att13.blif" line: 2, columns:64-65 [WR_UNREF01]: b0: Defined but not used
"att13.blif" line: 671, columns:12-13 [WR_UNREF01]: r7: Defined but not used
