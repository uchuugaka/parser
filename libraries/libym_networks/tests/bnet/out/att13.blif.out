.model att13
.inputs a b c d e f g h i j k l m n o p q r s t u v w x y z a0 b0 c0 d0 e0 f0 g0 h0 i0
.outputs j0 k0 l0 m0 n0 o0 p0 q0 r0 s0
.names h0 j0
0 1
.names h0 i1 c t6 k0
0--0 1
-0-0 1
--00 1
.names d i1
0 1
.names x3 d g1 u6 t6
0--0 1
-0-0 1
--00 1
.names b d1 v6 w6 x6 l0
0-00- 1
-00-0 1
.names b d1
0 1
.names h7 i v6
0- 1
-0 1
.names g g1 i7 f i1 j7 k7 w6
0--0--0 1
-0-0--0 1
--00--0 1
0---0-0 1
-0--0-0 1
--0-0-0 1
0----00 1
-0---00 1
--0--00 1
.names y6 x6
0 1
.names d1 e1 f1 m0
000 1
.names q1 r1 z e1
0-- 1
-0- 1
--0 1
.names g1 h1 i1 f1
-0- 1
0-0 1
.names d1 s1 t1 n0
000 1
.names r1 z s1
0- 1
-0 1
.names d0 q1 u1 v1 c0 w1 t1
0--0-- 1
-0-0-- 1
--00-- 1
0---0- 1
-0--0- 1
--0-0- 1
0----0 1
-0---0 1
--0--0 1
.names d1 s1 f2 o0
000 1
.names c0 g2 w1 e0 g1 h2 i2 f2
0--0--0 1
-0-0--0 1
--00--0 1
0---0-0 1
-0--0-0 1
--0-0-0 1
0----00 1
-0---00 1
--0--00 1
.names d1 s1 t2 p0
000 1
.names c0 u2 w1 f0 g1 v2 w2 t2
0--0--0 1
-0-0--0 1
--00--0 1
0---0-0 1
-0--0-0 1
--0-0-0 1
0----00 1
-0---00 1
--0--00 1
.names l1 m1 j3 k1 k3 l3 q0
-----0 1
---00- 1
000--- 1
.names u l1
0 1
.names q m1
0 1
.names t y s x r w j3
00---- 1
--00-- 1
----00 1
.names p k1
0 1
.names v r c3 q m3 s3 k3
0--0-0 1
-0-0-0 1
--00-0 1
0---00 1
-0--00 1
--0-00 1
.names v u m3 g0 n3 o3 l3
0--0-0 1
-0-0-0 1
--00-0 1
0---00 1
-0--00 1
--0-00 1
.names a0 v3 w3 r0
000 1
.names a v3
0 1
.names x3 g0 y3 h0 z3 w3
0--0- 1
-0-0- 1
--00- 1
0---0 1
-0--0 1
--0-0 1
.names a0 v3 h5 s0
000 1
.names i5 j5 k5 l5 p m5 n5 h5
0--0--0 1
-0-0--0 1
--00--0 1
0---0-0 1
-0--0-0 1
--0-0-0 1
0----00 1
-0---00 1
--0--00 1
.names c0 q1
0 1
.names a0 r1
0 1
.names c g1
0 1
.names x s y t j1 h1
----0 1
00--- 1
--00- 1
.names k1 l1 m1 n1 o1 p1 j1
0-0-0- 1
-00-0- 1
0--00- 1
-0-00- 1
0-0--0 1
-00--0 1
0--0-0 1
-0-0-0 1
.names v n1
0 1
.names r o1
0 1
.names w p1
0 1
.names c z1 d a2 u1
00-- 1
--00 1
.names d0 v1
0 1
.names c d x1 w1
0-0 1
-00 1
.names b2 c2 n1 m1 e2 z1
0-0-0 1
-00-0 1
0--00 1
-0-00 1
.names b2 c2 l1 k1 d2 a2
0-0-0 1
-00-0 1
0--00 1
-0-00 1
.names p1 o1 n1 m1 y1 x1
0-0-0 1
-00-0 1
0--00 1
-0-00 1
.names p u t y s x y1
00---- 1
--00-- 1
----00 1
.names y b2
0 1
.names t c2
0 1
.names p u s x r w e2
00---- 1
--00-- 1
----00 1
.names s x q v r w d2
00---- 1
--00-- 1
----00 1
.names e0 v1 g2
00 1
.names c0 r2 d0 z1 h2
00-- 1
--00 1
.names j2 d k2 i2
000 1
.names p1 o1 b2 c2 s2 r2
0-0-0 1
-00-0 1
0--00 1
-0-00 1
.names e0 j2
0 1
.names q1 l2 v1 m2 k2
0-0- 1
-00- 1
0--0 1
-0-0 1
.names w r y t q2 l2
----0 1
00--- 1
--00- 1
.names y t u p n2 m2
----0 1
00--- 1
--00- 1
.names k1 l1 m1 n1 o2 p2 q2
0-0-0- 1
-00-0- 1
0--00- 1
-0-00- 1
0-0--0 1
-00--0 1
0--0-0 1
-0-0-0 1
.names o2 p2 o1 p1 m1 n1 n2
0-0-0- 1
-00-0- 1
0--00- 1
-0-00- 1
0-0--0 1
-00--0 1
0--0-0 1
-0-0-0 1
.names s o2
0 1
.names x p2
0 1
.names p u q v s x s2
00---- 1
--00-- 1
----00 1
.names f0 j2 v1 u2
000 1
.names p q u v e3 f3 v2
-----0 1
0-0-0- 1
-0-00- 1
.names x2 d y2 w2
000 1
.names j2 v1 q1 e3
000 1
.names p1 o2 g3 o1 h3 i3 f3
-----0 1
---00- 1
000--- 1
.names f0 x2
0 1
.names k1 l1 m1 n1 z2 a3 y2
----00 1
0-0--0 1
-00--0 1
0--0-0 1
-0-0-0 1
.names e0 d0 c0 z2
0-- 1
-0- 1
--0 1
.names w s b3 r c3 d3 a3
0--0-0 1
-0-0-0 1
--00-0 1
0---00 1
-0--00 1
--0-00 1
.names b2 c2 b3
0- 1
-0 1
.names c2 b2 o2 p2 c3
0-0- 1
-00- 1
0--0 1
-0-0 1
.names p1 q1 v1 b2 c2 j2 p2 d3
-00--0- 1
0--0--0 1
0---0-0 1
.names y t g3
00 1
.names t y s x h3
00-- 1
--00 1
.names t w x y c0 d0 e0 i3
-0--0-- 1
--0-0-- 1
-0---0- 1
--0--0- 1
-0----0 1
--0---0 1
0--00-- 1
0--0-0- 1
0--0--0 1
.names c2 b2 o2 p2 o1 p1 m3
0-0-0- 1
-00-0- 1
0--00- 1
-0-00- 1
0-0--0 1
-00--0 1
0--0-0 1
-0-0-0 1
.names b2 c2 t3 s3
0-0 1
-00 1
.names d0 c0 r3 n3
0-- 1
-0- 1
--0 1
.names b z p3 o3
0-- 1
-0- 1
--0 1
.names x2 j2 r3
00 1
.names d c a0 q3 p3
0-00 1
-000 1
.names g0 x2 z2 q3
000 1
.names s x w v t3
--0- 1
---0 1
00-- 1
.names g0 u3
0 1
.names h0 x3
0 1
.names e4 o4 p4 q4 r4 y3
----0 1
000-- 1
0-00- 1
.names k1 a4 p b4 c4 z3
----0 1
00--- 1
--00- 1
.names t s r e4
0-- 1
-0- 1
--0 1
.names q k1 k o4
0-- 1
-0- 1
--0 1
.names x2 j2 f5 g5 p4
---0 1
000- 1
.names m1 p l q4
0-- 1
-0- 1
--0 1
.names s4 t4 u4 v4 w4 x4 r4
--0--0 1
0--0-0 1
-0-0-0 1
0---00 1
-0--00 1
.names q l4 r m4 n4 a4
----0 1
00--- 1
--00- 1
.names o1 m1 k4 b4
000 1
.names f0 e0 d4 m1 e4 f4 c4
0--0-0 1
-0-0-0 1
--00-0 1
0---00 1
-0--00 1
--0-00 1
.names o1 l k4 l4
000 1
.names c2 o2 m m4
000 1
.names n o c2 o2 n4
0-0- 1
-0-0 1
--00 1
.names t s k4
0- 1
-0 1
.names v1 q1 d4
0- 1
-0 1
.names c0 d0 e0 f0 g4 f4
----0 1
--00- 1
00-0- 1
.names o1 c2 o2 h4 g4
--00 1
00-0 1
.names t r g0 i4 h4
--0- 1
---0 1
00-- 1
.names t s j4 i4
0-- 1
-0- 1
--0 1
.names o1 m1 k j4
000 1
.names d0 c0 f5
00 1
.names q1 v1 j2 x2 g5
---0 1
0-0- 1
-00- 1
.names m1 k1 e5 s4
000 1
.names c2 o2 r t4
000 1
.names f0 e0 d4 c5 u4
0--0 1
-0-0 1
--00 1
.names m1 k1 d5 v4
000 1
.names c2 s o1 w4
000 1
.names f0 e0 d4 x2 j2 y4 x4
0--0-0 1
-0-0-0 1
--00-0 1
0---00 1
-0--00 1
--0-00 1
.names m e5
0 1
.names c0 d0 e0 f0 c5
--00 1
00-0 1
.names n d5
0 1
.names f0 d0 c0 c2 z4 y4
---0- 1
----0 1
000-- 1
.names k1 a5 b5 z4
000 1
.names o a5
0 1
.names s r q b5
0-- 1
-0- 1
--0 1
.names p r6 s6 i5
000 1
.names o2 o1 m1 j5
000 1
.names j2 b6 f5 g0 f0 q6 k5
000--- 1
---000 1
.names i0 l5
0 1
.names j2 b6 c6 m1 d6 e6 f6 m5
------0 1
000---- 1
---000- 1
.names l5 o5 n5
00 1
.names k r6
0 1
.names l5 t s6
0- 1
-0 1
.names g0 f0 b6
0- 1
-0 1
.names q1 v1 j2 q6
0-0 1
-00 1
.names m1 l6 q s5 m6 c6
----0 1
00--- 1
--00- 1
.names u3 x2 d6
0- 1
-0 1
.names o1 i6 j2 v1 j6 e6
----0 1
00--- 1
--00- 1
.names u3 x2 g6 f6
0-- 1
-0- 1
--0 1
.names p p5 k1 q5 r5 o5
0-0-0 1
-00-0 1
0--00 1
-0-00 1
.names m1 x5 o1 y5 z5 p5
0-0-0 1
-00-0 1
0--00 1
-0-00 1
.names r q w5 q5
0-- 1
-0- 1
--0 1
.names g0 j2 f5 q s5 t5 r5
-----0 1
---00- 1
000--- 1
.names r a6 w5 x5
0-- 1
-0- 1
--0 1
.names t s e5 y5
0-- 1
-0- 1
--0 1
.names s t a5 d5 z5
00-- 1
-00- 1
0--0 1
.names c2 o2 w5
00 1
.names c2 o2 o1 s5
000 1
.names o1 c2 o2 u5 t5
-0-0 1
0-00 1
.names m1 k e4 s r v5 u5
-----0 1
---00- 1
000--- 1
.names f0 g0 q1 v1 j2 x2 v5
00---- 1
--0-00 1
---000 1
.names l a6
0 1
.names r m4 n4 l6
--0 1
00- 1
.names v1 q1 c2 o2 n6 m6
0-0-0 1
-00-0 1
0--00 1
-0-00 1
.names s t a5 d5 i6
00-- 1
-00- 1
0--0 1
.names e0 c0 c2 o2 k6 j6
0-0-0 1
-00-0 1
0--00 1
-0-00 1
.names c0 d0 e0 h6 g6
--00 1
00-0 1
.names m1 l s5 h6
0-- 1
-0- 1
--0 1
.names m r s t c2 o2 k6
00---- 1
-00--- 1
-0-0-- 1
0---00 1
--0-00 1
---000 1
.names r s t o6 n6
---0 1
00-- 1
0-0- 1
.names t p6 o6
0- 1
-0 1
.names o2 o1 l p6
000 1
.names d i0 g1 u6
00- 1
0-0 1
-00 1
.names j h7
0 1
.names f i1 o7 h d7 p7 q7 i7
0--0--0 1
-0-0--0 1
--00--0 1
0---0-0 1
-0--0-0 1
--0-0-0 1
0----00 1
-0---00 1
--0--00 1
.names e c7 d7 h g1 g j7
0-0-0- 1
-00-0- 1
0--00- 1
-0-00- 1
0-0--0 1
-00--0 1
0--0-0 1
-0-0-0 1
.names h e l7 k7
0-0 1
-00 1
.names g1 z6 c a7 y6
0-0- 1
-00- 1
0--0 1
-0-0 1
.names h e i1 f7 z6
0--0 1
-0-0 1
--00 1
.names h e d b7 a7
0--0 1
-0-0 1
--00 1
.names c7 d7 d g7 f7
0--0 1
-0-0 1
--00 1
.names c7 d7 i1 e7 b7
0--0 1
-0-0 1
--00 1
.names h c7
0 1
.names e d7
0 1
.names i1 c7 d7 e7
00- 1
0-0 1
-00 1
.names d c7 d7 g7
00- 1
0-0 1
-00 1
.names e h o7
11 1
00 1
.names f i1 p7
0- 1
-0 1
.names i1 f h d7 q7
0-00 1
-000 1
.names d m7 e h c n7 l7
00---- 1
--00-- 1
----00 1
.names f m7
0 1
.names g n7
0 1
.names e h r7
0- 1
-0 1
.end
